demo.py

```diff
@@ -291,23 +292,23 @@ def py_fe_sqw(h, k, l, e, p):
 wsim = m.disp2sqw_eval(wsc, euobj.horace_disp, (scalefac), effective_fwhm)
 
 m.acolor('black')
-hf = m.plot(wsc); m.pl(wsim);
+hf = m.plot(wsc); m.pl(wsim)
 
 # Finally, we can repeat the simulation above but using the full instrument resolution
 # calculation using the `tobyfit` method of Horace
 
 # Calculate spectra with full instrument resolution convolution
-is_crystal = True;
-xgeom = [0,0,1];
-ygeom = [0,1,0];
-shape = 'cuboid';
-shape_pars = [0.01,0.05,0.01];
-wsc = m.set_sample(wsc, m.IX_sample(is_crystal, xgeom, ygeom, shape, shape_pars));
-ei = 40; freq = 400; chopper = 'g';
-wsc = m.set_instrument(wsc, m.merlin_instrument(ei, freq, chopper));
-disp2sqwfun = m.eval('@disp2sqw');
-kk = m.tobyfit(wsc);
-kk = kk.set_fun(disp2sqwfun, [euobj.horace_disp, [scalefac], intrinsic_fwhm]);
+is_crystal = True
```

Comment:
Can you remove this line (`is_crystal` is no longer used anywhere).

Also could you remove line 96/97 which defines `energy_range` since this is also not used anywhere. (The original script had a loop over the energies in `energy_range` and generates a cut at each energy for a total of 5 datasets to be fitted. This took a bit longer to run so I just cut it down to a single cut but forgot to remove the `energy_range` definition).

Reply:
I will do this now

Reply:
``` 
proj = {'u':[1,1,0], 'v':[-1,1,0], 'type':'rrr'}
energy_range = range(80, 160, 20)
w_fe = m.cut_sqw('datafiles/fe_cut.sqw', proj, [-3,0.05,3], [-1.05,-0.95], [-0.05,0.05], [70, 90])
```

Line 96 defining proj in the above appears to still be needed, why were you wanting it removed as well? @mducle